src/FGFDMExec.h

```diff
@@ -100,15 +100,15 @@ CLASS DOCUMENTATION
     from JSBSim. The state variables are used to drive the instrument displays
     and to place the vehicle model in world space for visual rendering:
 
-    @code{.cpp}
+    @code
     copy_to_JSBsim(); // copy control inputs to JSBSim
     fdmex->RunIC(); // loop JSBSim once w/o integrating
     copy_from_JSBsim(); // update the bus
     @endcode
 
     Once initialization is complete, cyclic execution proceeds:
 
-    @code{.cpp}
+    @code
```

Comment:
Please don't remove the `{.cpp}` elements
```suggestion
    @code{.cpp}
```